### PR TITLE
[Doc][Kommander] Ru/kommander 1.3.1 patch

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
Updated Release Notes for Kommander 1.3.1

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# Breaking changes
+
+In Kommander 1.3, unless a default NetworkPolicy existed prior to upgrading to 1.3, a default NetworkPolicy was created that allowed traffic originating only within your Projects namespace. Pods could not talk to Pods outside of that namespace. In some cases this is not a desired default state. Now no default NetworkPolicy is created and no blocking is done.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:31]: I got a little confused when reading this from the vantage point of somebody who might have started using Kommander with 1.3, and might not have been aware of this policy, or might have desired this policy.  With that in mind, I suggest this:

```suggestion
In Kommander 1.3, all new projects were created with a default NetworkPolicy that only allowed traffic originating within your Projects namespace.  This policy was also created in all existing projects that did not already have a NetworkPolicy when upgrading to Kommander 1.3.

The result of this default policy was that Pods could not talk to Pods outside of that namespace. In some cases this was not a desired default state.  As of Kommander 1.3.1, no default NetworkPolicy is created in new (or existing projects) and thus no traffic blocking is done by default.  
```

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# Breaking changes
+
+In Kommander 1.3, unless a default NetworkPolicy existed prior to upgrading to 1.3, a default NetworkPolicy was created that allowed traffic originating only within your Projects namespace. Pods could not talk to Pods outside of that namespace. In some cases this is not a desired default state. Now no default NetworkPolicy is created and no blocking is done. 
+If you made necessary changes to address this in Kommander 1.3, upgrading to Kommander 1.3.1 does not impact those changes.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:32]:
```suggestion
If you are upgrading from Kommander 1.3 and already made the necessary changes to address this issue (i.e. by adding a NetworkPolicy with the desired traffic rules), then upgrading to Kommander 1.3.1 will not impact those existing policies. 
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- Ensure pre-delete hook jobs are cleaned up.
+- Ensure kubectl deletes do not fail if resource already deleted.
+- Fix empty non-Konvoy cluster Platform Services tab.
+- Disable addons on disabled foundations.
+- All federated platform services/cluster platform services are now successfully removed when base addons are disabled.
+- Kubecost: Ensure kubectl deletes do not fail if resource is already deleted.
+- When self attaching a kommander cluster, the dex-k8s-authenticator configmap is no longer updated as this causes invalid links on the generate token page.
+- Fixes bug in Kommander UI where LDAP Root CA is not correct when saved.
+- Updates UI to only ship with needed dependencies.
+- Ensure finalizer is set on kubeaddons after re-enabling it.
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:53]:
```suggestion
```

I guess this was a bug fix on something that was added in 1.3.1 already so no need to include this one (not a bug fix of 1.3.0)

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- Add license delete mutation.
+- Replace license table with single license detail view.
+- Allow workspace namespace to be configurable.
+
+# Bug fixes
+
+- Ensure pre-delete hook jobs are cleaned up.
+- Ensure kubectl deletes do not fail if resource already deleted.
+- Fix empty non-Konvoy cluster Platform Services tab.
+- Disable addons on disabled foundations.
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:47]:
```suggestion
- Disable all platform services when Foundational Components are disabled in the UI.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- All federated platform services/cluster platform services are now successfully removed when base addons are disabled.
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:48]:
```suggestion
- All federated platform services are now successfully removed when the kubeaddons controller is disabled.
```

File: pages/dkp/kommander/1.3/release-notes/1.3.1/index.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+- Addon: 1.3.1-5
+- Chart: 0.15.3
+- kommander-federation (yakcl): 0.8.8
+- kommander-licensing (yakcl): 0.8.8
+- UI: 6.91.1
+- kommander-karma: 0.3.12
+- kubeaddons-catalog: 0.1.15
+- kommander-thanos: 0.1.16
+- kubecost: 0.5.3
```
Comment (gracedo) [inline pages/dkp/kommander/1.3/release-notes/1.3.1/index.md:65]:
```suggestion
- kubecost: 0.5.4
```

**Pull request merged**